tests/prefetch_related/models.py

```diff
@@ -313,3 +313,25 @@ class Flea(models.Model):
     )
     pets_visited = models.ManyToManyField(Pet, related_name="fleas_hosted")
     people_visited = models.ManyToManyField(Person, related_name="fleas_hosted")
+
+
+# Ticket 34791: Using to_attr for cache for singly related objects:
+
+
+class Country(models.Model):
+    label = models.CharField(max_length=50)
+
+
+class Pizza(models.Model):
+    label = models.CharField(max_length=50)
+    origin = models.ForeignKey(
+        Country, on_delete=models.CASCADE, null=True, related_name="pizzas"
+    )
+
+
+class Topping(models.Model):
+    pizza = models.ForeignKey(Pizza, on_delete=models.CASCADE, related_name="toppings")
+    label = models.CharField(max_length=50)
+    origin = models.ForeignKey(
+        Country, on_delete=models.CASCADE, related_name="toppings"
+    )
```

Comment:
In order to keep the test suite lean we avoid creating models (and tables) for each models. It would be appreciate if you could try reusing existing models for your test.